Source/WebKit/Platform/spi/ios/UIKitSPI.h

```diff
@@ -545,6 +545,10 @@ typedef enum {
 @property (readonly) NSString *_hostApplicationBundleIdentifier;
 @end
 
+@interface UIViewController (Private)
+- (/* nullable */ UIPresentationController *)_existingPresentationControllerImmediate:(BOOL)immediate effective:(BOOL)effective;
```

Comment:
Is the nullable commented out the recommended style for this SPI file?

Reply:
Yeah, we don't have other uncommented nullable declarations in this file, so we can't add any without adding all.